app/views/admin/settings/appearance/show.html.haml

```diff
@@ -27,5 +27,8 @@
           = fa_icon 'trash fw'
           = t('admin.site_uploads.delete')
 
+  .fields-group
+    = f.input :publish_button_text, wrapper: :with_block_label
```

Comment:
This should have a glitch-soc label, but block labels unfortunately don't support that :/